### PR TITLE
Added referrer parameter

### PR DESCRIPTION
**Pull request by bjorn-wo**
## Type of change

- [x ] Feature

## Description of change

Added a new parameter "referrer" where publisher can (optionally) insert the referrer domain if the ad is running within an iframe where the url can't be detected.

- contact email of the adapter’s maintainer
bandersson@wideorbit.com
- [x ] official adapter submission

### CODE REVIEW

File: integrationExamples/gpt/pbjs_example_gpt.html [hunk 1]
```diff
@@ -343,6 +343,13 @@
                         params: {
                             inScreen: 'ggumtest' // REQUIRED str Tracking Id
                         }
+                    },
+                    {
+                        bidder: 'wideorbit',
+                        params: {
+                            pbId: 123, // REQUIRED Publisher Id,
+                            pId: 123456 // REQUIRED Placement Id
+                        }
```
Comment (matthewlane) [inline integrationExamples/gpt/pbjs_example_gpt.html]: `pbjs_example_gpt.html` already includes the same wideorbit params further up the page, are these needed or can they be removed?

File: src/adapters/wideorbit.js [hunk 2]
```diff
@@ -1,213 +1,215 @@
-var bidfactory = require('../bidfactory.js'),
```
Comment (matthewlane) [inline src/adapters/wideorbit.js]: The line endings in this commit are CRLF (Windows-style), but the existing wideorbit commit uses LF (Unix-style) line endings. Can you keep LF line endings so the diff is easier to read?

File: integrationExamples/gpt/pbjs_example_gpt.html (hunk 1 above)
Comment (bjorn-wo) [inline integrationExamples/gpt/pbjs_example_gpt.html]: Thank you for spotting this. It looks like there was a problem when mergin in the latest code.

**Comment by aneuway2** [comment]
@bjorn-wo can you add information on the new `referrer` parameter to the [docs repo](https://github.com/prebid/prebid.github.io) ?

http://prebid.org/dev-docs/bidders.html#wideorbit

**Pull request merged**